docs/backups.md

```diff
@@ -487,6 +487,14 @@ $ kubectl apply -f deploy/backup/restore.yaml
     EOF
     ```
 
+<a name="backup-pitr-binlog-gaps"></a>
+
+Take into account, that Operator monitors the binlog gaps detected by
+binlog collector, if any. If backup contains such gaps, the Operator marks such
+backup as "not suitable for consistent PITR". Attempts to restore such backup 
+with point-in-time recovery will be blocked, and the restore operation will be
+marked as failed.
```

Comment:
> the Operator marks such backup as "not suitable for consistent PITR
this marking is done by a special condition called `PITRReady` and can be found in `.status.conditions` field of backup. Not having this condition means that operator didn't detect any gaps.

> and the restore operation will be marked as failed.
if you want to force the restore anyway, you need to annotate restore object with `percona.com/unsafe-pitr: "true"`